docs/design.html

```diff
@@ -136,7 +136,7 @@
     Kafka supports this with an efficient batching format. A batch of messages can be clumped together compressed and sent to the server in this form. This batch of messages will be written in compressed form and will
     remain compressed in the log and will only be decompressed by the consumer.
     <p>
-    Kafka supports GZIP, Snappy and LZ4 compression protocols. More details on compression can be found <a href="https://cwiki.apache.org/confluence/display/KAFKA/Compression">here</a>.
+    Kafka supports GZIP, Snappy and LZ4 compression protocols. Compression related config properties can be found in the <a href="#producerconfigs">Producer configs</a> section.
```

Comment:
Hmm, not sure about this. Compression related config properties can be found in the producer and broker configs, right?

Reply:
@ijuma  yes.  mentioned broker configs in the compression docs.

Reply:
@ijuma pinging for review